### DISCUSSION TITLE
why not in Boostrap ? Detail Row

### DISCUSSION THREAD

**Discussion by hemant17** [Power-Components/livewire-powergrid#717] [Ideas]
# ⚡ PowerGrid ⚡ Feature Request

## Summary
Detail Row is an awesome Feature why not available in Bootstrap only in Tailwind?

Brief description of what is at stake.

### Why is this needed?
I think no need to explain why it's needed.

**Comment by luanfreitasdev** [comment]
Hello. Currently not available for bootstrap. If you use bootstrap in your project, feel free to send us a PR
